### PR TITLE
#43. StrictXMLTest.passesValidXmlWithNetworkProblems fixed

### PR DESCRIPTION
**Pull request by alevohin**
Issue #43. StrictXMLTest.passesValidXmlWithNetworkProblems fixed

### CODE REVIEW

**Comment by alevohin** [comment]
@dmarkov build failed in appveyor due to reasons mentioned in https://github.com/jcabi/jcabi-xml/pull/42

**Comment by dmarkov** [comment]
Thanks, someone will review your pull request soon

**Comment by alevohin** [comment]
@dmarkov @yegor256 we need someone to review PR.

**Comment by dmarkov** [comment]
@carlosmiranda please review, thanks

**Comment by carlosmiranda** [comment]
@rultor Mind if we merge?

**Comment by carlosmiranda** [comment]
@rultor Let's merge

File: src/test/java/com/jcabi/xml/StrictXMLTest.java [hunk 1]
```diff
@@ -168,27 +175,33 @@ public Void call() throws Exception {
...
+                    final int attempt = counter.incrementAndGet();
+                    if (attempt == 1 || attempt == 2) {
+                        throw new SocketException(
+                            String.format("Attempt #%s failed", attempt)
+                        );
+                    } else {
+                        return null;
+                    }
+                }
+            }).when(validator).validate(Mockito.any(Source.class));
```
Comment (yegor256) [inline src/test/java/com/jcabi/xml/StrictXMLTest.java]: I would put this closing bracket to the new line, see http://www.yegor256.com/2014/10/23/paired-brackets-notation.html

File: src/test/java/com/jcabi/xml/StrictXMLTest.java [hunk 2]
```diff
@@ -168,27 +175,33 @@ public Void call() throws Exception {
...
+                // @checkstyle IllegalThrowsCheck (5 lines)
+                @Override
+                public Void answer(final InvocationOnMock invocation)
+                    throws Throwable {
+                    final int attempt = counter.incrementAndGet();
+                    if (attempt == 1 || attempt == 2) {
+                        throw new SocketException(
+                            String.format("Attempt #%s failed", attempt)
+                        );
+                    } else {
```
Comment (yegor256) [inline src/test/java/com/jcabi/xml/StrictXMLTest.java]: no need to do `else` here. just return null.

**Comment by yegor256** [comment]
@alevohin a few code formatting comments above

**Comment by alevohin** [comment]
@carlosmiranda @yegor256 thanks! Please have a look one more time.

CI builds failed due to:
1) appveyor because of #44
2) travis-ci got problem
> GET request of: org/hibernate/hibernate-validator/5.1.0.Final/hibernate-validator-5.1.0.Final.jar from central failed

Maybe there is some way to restart CI build withour new push?

**Comment by yegor256** [comment]
@rultor merge it again

**Comment by yegor256** [comment]
@alevohin the best way to restart Travis is to push something to the branch :) just change one letter somewhere and push

**Comment by dmarkov** [comment]
@carlosmiranda Many thanks! **18 mins** were added to your account in Transaction ID `50566048`. added +18 to your rating, now it is equal to [+1667](http://www.netbout.com/b/35023?open=rating)

**Comment by dmarkov** [comment]
@rultor deploy

**Pull request merged**